src/DssSpell.sol

```diff
@@ -16,439 +16,126 @@
+        DssExecLib.setIlkAutoLineParameters("UNIV2WBTCETH-A",     20_000_000,   3_000_000, 8 hours);
+        DssExecLib.setIlkAutoLineParameters("UNIV2USDCETH-A",     50_000_000,   5_000_000, 8 hours);
+        DssExecLib.setIlkAutoLineParameters("UNIV2DAIUSDC-A",     50_000_000,  10_000_000, 8 hours);
+        DssExecLib.setIlkAutoLineParameters("UNIV2ETHUSDT-A",     10_000_000,   2_000_000, 8 hours);
+        DssExecLib.setIlkAutoLineParameters("UNIV2LINKETH-A",     20_000_000,   2_000_000, 8 hours);
+        DssExecLib.setIlkAutoLineParameters("UNIV2UNIETH-A",      20_000_000,   3_000_000, 8 hours);
+        DssExecLib.setIlkAutoLineParameters("UNIV2WBTCDAI-A",     20_000_000,   3_000_000, 8 hours);
+        DssExecLib.setIlkAutoLineParameters("UNIV2AAVEETH-A",     20_000_000,   2_000_000, 8 hours);
+        DssExecLib.setIlkAutoLineParameters("UNIV2DAIUSDT-A",     10_000_000,   2_000_000, 8 hours);
+
+        // ----------------------------- Stability Fee updates --------------------------
+        DssExecLib.setIlkStabilityFee("ETH-A", THREE_POINT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("ETH-B", NINE_PCT, true);
+        DssExecLib.setIlkStabilityFee("ETH-C", ONE_PCT, true);
+        DssExecLib.setIlkStabilityFee("WBTC-A", THREE_POINT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("LINK-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("YFI-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNI-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("AAVE-A", TWO_PCT, true);
+        DssExecLib.setIlkStabilityFee("BAT-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("RENBTC-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2DAIETH-A", TWO_POINT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", THREE_POINT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCETH-A", THREE_POINT_FIVE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2UNIETH-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2ETHUSDT-A", FOUR_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2LINKETH-A", THREE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2AAVEETH-A", THREE_PCT, true);
+        DssExecLib.setIlkStabilityFee("UNIV2DAIUSDT-A", TWO_PCT, true);
+
+        // ----------------------------- UNIV2DAIUSDC-A SF and CR -----------------------
+        DssExecLib.setIlkLiquidationRatio("UNIV2DAIUSDC-A", 10200);
+        DssExecLib.setIlkStabilityFee("UNIV2DAIUSDC-A", POINT_FIVE_PCT, true);
+
+        // ----------------------------- ETH Auction Params -----------------------------
+        Fileable(MCD_DOG).file("ETH-A", "hole", 30_000_000 * RAD);
+        Fileable(MCD_DOG).file("ETH-B", "hole", 15_000_000 * RAD);
+        Fileable(MCD_DOG).file("ETH-C", "hole", 20_000_000 * RAD);
+        Fileable(DssExecLib.getChangelogAddress("MCD_CLIP_CALC_ETH_B")).file(
+            "step", 60 seconds
```

Comment:
Is `seconds` necessary here?

Reply:
not sure, but has been used in previous spells as well

Reply:
no, just leads to a clarity in the units used.